ui/nuxeo-aggregation/nuxeo-checkbox-aggregation.js

```diff
@@ -73,6 +142,88 @@ import { AggregationBehavior } from './nuxeo-aggregation-behavior.js';
     static get is() {
       return 'nuxeo-checkbox-aggregation';
     }
+
+    static get properties() {
+      return {
+        _buckets: {
+          type: Array,
+          value: () => [],
+        },
+        /**
+         * Make the element collapsible
+         */
+        collapsible: {
+          type: Boolean,
+          value: false,
+        },
+
+        /**
+         * Label of the element
+         */
+        label: {
+          type: String,
+          value: '',
+        },
+
+        _show: {
+          type: Boolean,
+          value: false,
+        },
+
+        _slicedBuckets: {
+          type: Array,
+          value: () => [],
+        },
+
+        _visibleBuckets: {
+          type: Array,
+          value: () => [],
+        },
+
+        /**
+         * If collapsible, the default visible items.
+         */
+        visibleItems: {
+          type: Number,
+          value: 8,
+        },
+      };
+    }
+
+    _checkCollapse(buckets) {
+      if (!this.collapsible && buckets.length) {
+        this._visibleBuckets = buckets;
+        return true;
+      }
+      return false;
+    }
+
+    _setBuckets(buckets) {
+      if (this.collapsible && buckets.length) {
+        this._buckets = buckets;
+        this._slicedBuckets = this._buckets.slice(0, this.visibleItems);
+        this._visibleBuckets = this._slicedBuckets;
+        return true;
+      }
+    }
```

Comment:
I guess we can simplify this logic here, since we don't need to explicitly keep both arrays of buckets.
Probably we can achieve the same results with the same method and also simplifying/reducing the DOM by using only `_visibleBuckets`, for instance 😉